meta-iot2000-bsp/README.md

```diff
@@ -23,8 +23,8 @@ Building the meta-iot2000 BSP Layer
 ===================================
 
 This uses Yocto 2.2 (Morty) with the 4.4 Linux kernel provided by Yocto and
-the meta-intel layer. The build has been tested successfully on Debian 8 and
-OpenSuse 42.2, but other recent distros are expected to work as well.
+the meta-intel layer. The build has successfully been tested on Debian 8,
```

Comment:
Are you sure about this reordering of words? I'm not a native speaker, just unsure.